system-contracts/contracts/Constants.sol

```diff
@@ -105,7 +105,7 @@ uint256 constant STATE_DIFF_ENTRY_SIZE = 272;
 
 /// @dev While the "real" amount of pubdata that can be sent rarely exceeds the BLOB_SIZE_BYTES * MAX_NUMBER_OF_BLOBS, it is better to
 /// allow the operator to provide any reasonably large value in order to avoid unneeded constraints on the operator.
-uint256 constant MAX_ALLOWED_PUBDATA_PER_BATCH = 520000;
+uint256 constant MAX_ALLOWED_PUBDATA_PER_BATCH = 1_560_000;
```

Comment:
I don't understand the current purpose of this constant and its usage. `BLOB_SIZE_BYTES * MAX_NUMBER_OF_BLOBS` is `126976 * 6` is `761856`. While `MAX_ALLOWED_PUBDATA_PER_BATCH` is more than twice larger number.

The check where `MAX_ALLOWED_PUBDATA_PER_BATCH` is used is here: 
- https://github.com/matter-labs/era-contracts/blob/zk-6-blobs/system-contracts/contracts/L1Messenger.sol#L298

But we also have this (stronger) check:
- https://github.com/matter-labs/era-contracts/blob/zk-6-blobs/system-contracts/contracts/PubdataChunkPublisher.sol#L22

So I think this constant should be removed as it doesn't really constraint anything and also the point of having check that allows to provide more than twice more pubdata doesn't makes much sense tbh.

Reply:
for pubdata we add in a lot of data to verify the state diff compression